quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/QuarkusPropertiesDistTest.java

```diff
@@ -132,6 +133,25 @@ void testUnknownQuarkusBuildTimePropertyApplied(LaunchResult result) {
                 .body(containsString("jvm_gc_"));
     }
 
+    @Test
+    @Launch({ "start", "--http-enabled=true", "--hostname-strict=false", "--config-keystore=keystore" })
+    @Order(10)
+    void testMissingSmallRyeKeyStoreProperty(LaunchResult result) {
+        CLIResult cliResult = (CLIResult) result;
+        cliResult.assertError("The config property smallrye.config.source.keystore.kc-default.password is required but it could not be found in any config source");
```

Comment:
This error message brings me to a question whether we shouldn't have some custom validation with more user friendly message. Also, it seems SmallRye doesn't check whether the keystore file actually exists. Maybe we could handle that too. But both are probably just a follow up. WDYT?

CC @pedroigor

Reply:
Actually, even the config file (`-cf`) behaves the same – no error when file does not exist.